source/common/vue/form/FormBuilder.vue

```diff
@@ -15,7 +15,10 @@
       </div>
       <fieldset>
         <!-- First, let's do some setup of the fieldset -->
-        <div class="form-header">
+        <div
+          class="form-header"
+          v-bind:class="{ 'space-between': fieldset.title === 'Application language' }"
```

Comment:
Is this really only necessary for a single field? This seems brittle, especially should we ever decide to change its name … feel free to explain what you want to achieve.